app/helpers/account_helper.rb

```diff
@@ -2,4 +2,8 @@ module AccountHelper
   def govuk_account_auth_enabled?
     ENV["FEATURE_FLAG_GOVUK_ACCOUNT"] == "enabled"
   end
+
+  def single_page_notifications_feature_enabled?
+    ENV["SINGLE_PAGE_NOTIFICATIONS_FEATURE"] == "enabled"
+  end
```

Comment:
I don't think we need a separate feature flag for this, as we're not going to release these things independently.

Reply:
Do you mean just use: `FEATURE_FLAG_GOVUK_ACCOUNT` ?